examples/python/queries/nearest_neighbor_interpolation.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import _dummy_objects
+
+# Load a map
+your_map = _dummy_objects.example_map()
+
+# Declare the point to query [in map frame]
+query_point = np.array([0.4, .5, 0.6])
```

Comment:
Should probably be `0.5` for consistency.

Reply:
Thanks :)